src/compass/utils/lut.py

```diff
@@ -76,16 +77,16 @@ def cumulative_correction_luts(burst, dem_path,
     output_path = f'{scratch_path}/corrections'
     os.makedirs(output_path, exist_ok=True)
     data_list = [geometry_doppler, bistatic_delay.data, az_fm_mismatch.data,
-                 tide_rg]
+                 tide_rg, ionosphere]
     descr = ['geometrical doppler', 'bistatic delay', 'azimuth FM rate mismatch',
-             'slant range Solid Earth tides']
+             'slant range Solid Earth tides', 'ionospheric delay']
```

Comment:
Let's specify that the ionosphere is along the LOS.

Reply:
Good idea. For consistency, what do you think about adding `azimuth` or `slant range` for the first three corrections? i.e. `geometrical doppler`, `bistatic delay`, and `azimuth FM rate mismatch`

Reply:
I agree, it is a good idea.